### PR TITLE
#25_client_carousel_component

### PR DESCRIPTION
**Pull request by Roxha-V**
implementation of carousel component for client- carousel section. Swiper library is used. New tasks (as a propousal) may be: to edit carousel style, and differentiating the category of component and section in this and other elements of the app.

### CODE REVIEW

File: public/img/logo6.jpg
Comment (devPalter) [inline public/img/logo6.jpg:1]: estos logos van a ser usados?

File: app/components/clients_carousel/index.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import React, { useEffect, useRef, useState } from "react";
+import { Autoplay, Navigation, Pagination } from "swiper/modules";
+import { Swiper, SwiperSlide } from "swiper/react";
+
+function Clients_Carousel({ data }) {
+  return (
+    <div className="w-full h-full">
```
Comment (devPalter) [inline app/components/clients_carousel/index.jsx]: falta añadir id de la seccion para navegación interna

**Review by devPalter — CHANGES_REQUESTED** [review]

**Comment by devPalter** [comment]
Luego de las correciones hacer un rebase de los commits en 1 solo

File: public/img/logo6.jpg
Comment (Roxha-V) [reply public/img/logo6.jpg:1]: logo images are not necessary, they were used as an example to finish the structure of the carousel. They will be deleted from the project.

**Comment by Roxha-V** [comment]
The revised changes are resolved, rebase is made to develop and all the commits are merged into one. Last revision is requested to be able to merge the branches.

File: app/page.js [hunk 2]
```diff
@@ -2,15 +2,34 @@ import React from "react";
...
+];
 export default function Home() {
   return (
-    <>
-      <HeroComponent />
-      <AboutUs />
-      <Services />
-      <FloatingBtn />
-    </>
+    <main className="min-h-screen justify-between">
```
Comment (devPalter) [inline app/page.js]: why? is necesary?

**Review by devPalter — APPROVED** [review]

**Pull request merged**